utils/zerovec/derive/src/utils.rs

```diff
@@ -275,3 +278,43 @@ pub fn extract_attributes_common(
 
     Ok(attrs)
 }
+
+pub(crate) fn find_validate_with_path(
```

Comment:
issue: This should be a part of the attribute parsing above in `extract_attributes_common`